index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav>
+        <header class="toolbar">
+          <div class="logo">
+            <div class="placeholder">
+              <h3>My Logo</h3>
+            </div>
+          </div>
+
+          <div class="normal-button">
+            <div id="icon-menu">
+              <div class="menu-line"></div>
+              <div class="menu-line"></div>
+              <div class="menu-line"></div>
+            </div>
+          </div>
+        </header>
+      </nav>
+
+      <section class="bg-color-1">
+        <div class="bg-image-1"></div>
+        <div class="headline">
+          <h1 class="headline-inside">I’m Mariane Glad to see you!</h1>
+          <p class="headline-inside2">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <h3 class="headline-inside3">Let's connect</h3>
+          <ul class="social-media">
+            <li><img src="./assets/twitter.svg" alt="twitter" /></li>
+            <li><img src="./assets/linkedin.svg" alt="linkedIn" /></li>
+            <li><img src="./assets/M.svg" alt="icon" /></li>
+            <li><img src="./assets/cat.svg" alt="icon" /></li>
+            <li><img src="./assets/victory.svg" alt="icon" /></li>
+          </ul>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that the social icons align with the `h3` tag "Let's connect"  to the left as provided in the design template.

Your Implementation   |  Figma Design

:-------------------------:|:-------------------------:

![Screenshot from 2022-02-16 09-23-06](https://user-images.githubusercontent.com/58692590/154225441-5f0ab7b6-b0a5-4030-93fb-09de842836a6.png) | ![Screenshot from 2022-02-16 09-29-19](https://user-images.githubusercontent.com/58692590/154225463-4f1b9e9b-abbb-4118-a5b8-726409d6736a.png)